randomImages/app.js

```diff
@@ -0,0 +1,6 @@
+var imagesArray = ["https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900", "https://source.unsplash.com/1600x900"];
```

Comment:
use the API for for unsplash to generate random images

by changing the src of the mage you will be able to get random picture from unsplash

```
let url = ("https://source.unsplash.com/random");
let image = document.querySelector(".img");
image.setAttribute("src", url);
```